src/main/java/liubomyr/stepanenko/bookstore/security/CustomUserDetailsService.java

```diff
@@ -0,0 +1,20 @@
+package liubomyr.stepanenko.bookstore.security;
+
+import liubomyr.stepanenko.bookstore.repository.user.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class CustomUserDetailsService implements UserDetailsService {
+    private final UserRepository userRepository;
+
+    @Override
+    public UserDetails loadUserByUsername(String email) throws UsernameNotFoundException {
```

Comment:
make exception unchecked. 
```suggestion
    public UserDetails loadUserByUsername(String email) {
```